### PR TITLE
Enable formatting in std.big.Int.format

### PR DESCRIPTION
**Pull request by joachimschmidt557**

### CODE REVIEW

File: lib/std/math/big/int.zig [hunk 1]
```diff
@@ -381,14 +381,15 @@ pub const Int = struct {
         return if (d < base) d else return error.DigitTooLargeForBase;
     }
 
-    fn digitToChar(d: u8, base: u8) !u8 {
+    fn digitToChar(d: u8, base: u8, uppercase: bool) !u8 {
         if (d >= base) {
             return error.DigitTooLargeForBase;
         }
 
+        const a = if (uppercase) 'A' else 'a';
```
Comment (LemonBoy) [inline lib/std/math/big/int.zig:389]:
```suggestion
        const a: u8 = if (uppercase) 'A' else 'a';
```

Comment (joachimschmidt557) [reply lib/std/math/big/int.zig:389]: Oops, will fix that.

**Comment by LemonBoy** [comment]
You should also update all the `toString` callsites (eg. in `translate_c.zig`)

**Comment by joachimschmidt557** [comment]
Yup, will do

**Comment by joachimschmidt557** [comment]
The tests need to be updates as well of course

**Pull request merged**